imagepicker/src/main/res/layout/ef_imagepikcer_snackbar.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="@dimen/ef_height_snackbar"
+    android:background="@color/ef_black_alpha_aa"
```

Comment:
There's no reason to move this properties to here, not to mention changing `merge` to `RelativeLayout` means we have nested `RelativeLayout`